### PR TITLE
Recipe app

### PR DESCRIPTION
**Pull request by apt2tell**
## Using Ruby gems as a software packages system to build a `Recipe App` on ror

> I and @arabiu033 accomplished the following tasks in this  project:

- Implemented a login page.
- Added a registration page.
- Implemented a food list (with all CRUD implementation, except for 'update').
- Added recipes list (with all CRUD implementation, except for 'update').
- Added general shopping list view (all missing food for all your recipes and total price).
- Implemented a list of all public recipes from other users with their names and total prices.
- Updated `README.md` file

### CODE REVIEW

File: app/views/foods/_create.html.erb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<%= form_with(model: food, url: foods_path(:recipe => @recipe)) do |form| %>
```
Comment (rloterh) [inline app/views/foods/_create.html.erb:1]: - [ ] OPTIONAL
I think we can improve upon the UI of the food form by ensuring that the label associated with a particular input field are closer.

**Food form [Label apart]:**
image

File: README.md [hunk 2]
```diff
@@ -1,24 +1,166 @@
...
+
+If you like this project, kindly give it a star⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (rloterh) [inline README.md:164]: - [ ] MIT license link throws a 404-page request error when visited. Kindly endeavor to fix it.

File: README.md [hunk 3]
```diff
@@ -1,24 +1,166 @@
...
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [Aminu Rabiu](https://github.com/arabiu033/recipes_app.git)
+- LinkedIn: [Aminu Rabiu](https://www.linkedin.com/in/arabiu033/)
+
+👤 **Author2**
```
Comment (rloterh) [inline README.md]: - [ ] OPTIONAL
Generally, the subheading text _Author1_ and _Author2_ or similar should be replaced with your respective full names.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Aminu & Okei, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦  🏃  🏃‍♀️ 

## Highlights 👍  
- Authentication with Devise gem properly implemented
- App schema closely follows given ERD
- `N + 1` properly handled
- Design views matches given wireframe
- Unit test framework properly setup
- Rspec specs running correctly
- Good ReadMe
- Reviewed projects in turns

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly check the inline comments marked **[OPTIONAL]** under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @Okechukwu-muokwugwo and @arabiu033, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👍 You did a great job working on the changes requested by the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**